Constitution;Outlines how the government works and other important things.html

```diff
@@ -104,6 +104,11 @@ <h2>States of emergencies</h2>
     In the event that quick decision making is essential in an emergency, the ministry may have a vote to give the
     royalty temporary power
     to make temporary decisions.
+    </p>
+<h4>Minister of Mining </h4>
+<p>
+   This minister will take care of the gold problems people can apply to be a miner and get maxed picks if the picks break more than 7 times a week the miner will be fired
```

Comment:
Grammar is off
```suggestion
   This minister will take care of the gold problems.
   People can apply to be a miner and get maxed picks.
   If the picks break more than 7 times in a week, the miner will be fired.
```

Also, I don't think you quite understand what a Minister is. A Minister is someone who is elected in the monthly elections, Ministers are not fired and you don't "apply" to be a Minister, you run in the elections.